modules/panorama/variables.tf

```diff
@@ -46,6 +46,8 @@ variable "public_static_ip" {
   default     = null
 }
 
+}
```

Comment:
@Avinash-jayakumar those two lines have to be deleted as well. There was some issue with the "suggestion", did not cover enough lines.

Reply:
Please check now. commited . 4bfd2d35f25b216e0731beb5613fbd6104589415